src/lib/host_functions/hashing.rs

```diff
@@ -0,0 +1,62 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
You can call the folder `ops`. I believe this is the convention used in rusk-vm; so it would be `src/lib/ops/hashing.rs`.

Reply:
kk Sounds good